opencti-platform/opencti-graphql/src/http/httpChatbotProxy.ts

```diff
@@ -0,0 +1,92 @@
+import nconf from 'nconf';
+import axios from 'axios';
+import type Express from 'express';
+import { createAuthenticatedContext } from './httpAuthenticatedContext';
+import { getEntityFromCache } from '../database/cache';
+import { ENTITY_TYPE_SETTINGS } from '../schema/internalObject';
+import { getEntrepriseEditionActivePem } from '../modules/settings/licensing';
+import { logApp } from '../config/conf';
+import type { BasicStoreSettings } from '../types/settings';
+import { setCookieError } from './httpUtils';
+
+export const getChatbotProxy = async (req: Express.Request, res: Express.Response) => {
+  try {
+    const context = await createAuthenticatedContext(req, res, 'chatbot');
+    if (!context.user) {
+      res.sendStatus(403);
+      return;
+    }
+
+    const chatbotUrl = nconf.get('ai:chatbot:url');
+    const isChatbotEnabled = nconf.get('ai:chatbot:enabled');
+    const settings = await getEntityFromCache<BasicStoreSettings>(context, context.user, ENTITY_TYPE_SETTINGS);
+    const license_pem = getEntrepriseEditionActivePem(settings.enterprise_license);
+    if (!isChatbotEnabled || !license_pem) {
```

Comment:
We need to check if license is valid also no ? I mean if a platform has an expired license the request to chatbot will be send. I think you need to add isEnterpriseEdition from ee.ts.

Not sure but maybe also a check on FF ?